### ISSUE TITLE
Use with containerd

### ISSUE THREAD

**Issue by Crayeth** [hasheddan/k8scr#7]
Hi, it seems that this project has a dependency on docker that I cannot run in my organisation. Any chance for supporting containerd? My use-case is local crossplane package development on m1 Mac.

**Comment by hasheddan** [comment]
@Crayeth yep that should be possible :) Feel free to share any additional details about your setup so I can be sure the changes accommodate your use case

**Comment by Crayeth** [comment]
@hasheddan Nice! Right now Im using Colima w. containerd runtime to run k8s locally on my m1 macbook. We cannot use docker 4 or rancher desktop (yet). I want an easy way to build and test crossplane packages without depending on remote registries. I've tried a couple of things but not yet successfully. 

This tool seems like it would take away some of the pain. Running a registry in-cluster and pushing and pulling from it would really help.
